README.md

```diff
@@ -12,6 +12,33 @@ https://docs.ssx.id
 - Server side under [`./packages/ssx-server`](./packages/ssx-server).
 
 ## Useful Commands
-- `yarn build` - Build all packages
-- `yarn run-examples` - Concurrently run the example server and app found in `./examples`
-- `yarn clean && npx lerna clean -y` - Remove all build artifacts and node_modules
+- `npm run build` - Build all packages
+- `npm run examples` - Concurrently run the example server and app found in `./examples`
+- `npm run clean` - Remove all build artifacts and node_modules
+
+## Docker
+
+SSX ships with a [Docker Compose](https://docs.docker.com/compose/) configuration 
+for setting up a local development and testing environment with SSX server and 
+our example dapp. Make sure you have 
+[Docker and Compose installed](https://docs.docker.com/compose/install/), then 
+create a `.env` file in the project root:
+
+```
+ssx_listenPort=8443 # ssx server will be available on http://localhost:8443 
+ssx_signingKey= # session secret for ssx server
+ssx_providers__metrics__apiKey= # ssx metrics api key https://app.ssx.id
```

Comment:
Are the double underscores on purpose?

Reply:
Yeah, the `rc` module has this convention for overloading nested config properties thru environment vars `<appname>_<toplevel>__<nested>`

Reply:
Okie dokie, I was surprised it was a concern in javascript considering it uses camelCase!